pkg/yurthub/proxy/remote/remote.go

```diff
@@ -101,6 +103,21 @@ func (rp *RemoteProxy) modifyResponse(resp *http.Response) error {
 				h.Add("Transfer-Encoding", "chunked")
 			}
 		}
+
+		// 404 Not Found: The CRD may have been unregistered and should be updated locally as well
+		if resp.StatusCode == http.StatusNotFound {
```

Comment:
I think it's not need to cache CRD kind info on node local when CRD is not exist in cluster.